CameraServer/Helpers/Camera.cs

```diff
@@ -34,10 +34,16 @@ public async Task<CameraImage> GetImageAsync()
 
         public bool IsValidToken(string token)
         {
-            if(information == null || information.Token == null)
+            if (information == null || information.Token == null)
                 return false;
 
             return TokenHelper.GetToken(information.Id) == information.Token;
         }
+
+        public async void SetPreview()
+        {
+            await Task.CompletedTask;
+            Information.Preview = Convert.ToBase64String(GetImageAsync().Result.Bytes);
```

Comment:
du använder en async metod men sen kör du en async funktion i den synkront och väntar in resultatet...? Använd await framför GetImageAsync() istället